docs/backend/upgrading/upgrade-to-volto.md

```diff
@@ -10,54 +10,54 @@ html_meta:
+When creating a new site in Plone 6 you have the option to choose between creating a Volto site and a Classic Plone site.
+This is because there are some non-trivial differences between the configuration of a Classic Plone and a Volto site.
 
-1.  Why would someone want to upgrade (migrate?) to Volto versus staying with Classic UI?
-1.  What is the target audience?
-1.  What are the pre-requisites?
-1.  Other?
+This document discusses these differences and tells admins and developers how to modify their existing Plone site to be compatible with Volto.
+Pre-requisites: The site needs to be upgraded to Plone 6 before
 
-If you provide bullet points, I can write up the narrative and push to this PR.
-```
+Plone provides a form `/@@migrate_to_volto` that allows you to run all the required changes to your existing site to make it compatible with Volto.
+You can access this form in the browser when you are logged-in as an administrator by opening `http://localhost:8080/Plone/@@migrate_to_volto` (where `localhost` is your server, `8080` the port that Plone is running on and `Plone` is the name of the Plone instance).
+
+After upgrading an existing site to Plone 6 (see {doc}`v60`) there will also appear a message **You can prepare your site for Volto, the default frontend of Plone 6!** with a link to that form.
 
-Upgrading an existing site to run with Volto requires several changes to your existing content stored in the backend.
-The form `/@@migrate_to_volto` allows you to run all these steps directly.
-After you upgraded your existing Database to Plone 6 there also is a message **You can prepare your site for Volto, the default frontend of Plone 6!** with a link to the form `localhost:8080/Plone/@@migrate_to_volto`.
 
-```{todo}
-The previous sentence lacks context.
-When did I upgrade my existing database?
-Is there documentation that describes the database upgrade process?
-```
 
 ```{warning}
 Test all migrations thoroughly before applying them on a production environment!
 
-A migrated site will still work with Plone Classic UI, but it will behave differently.
-For example, editors can only effectively work with the content using Volto.
+A site that is make compatible with Volto will be accessible with Plone Classic UI, but it will behave differently.
+For example, editors can only effectively work with the content using Volto because html is no longer editable in the TinyMCE editor.
 ```
 
-
 ```{todo}
 `{py:mod}` does not link to the Python modules (see https://deploy-preview-1254--6-dev-docs-plone-org.netlify.app/backend/upgrading/upgrade_to_volto.html).
 
 We could either use inline literals as shown in the suggestion, or we could link to their Glossary entry (or create one), GitHub repo, PyPI project, or relevant chapter in the docs. Which do you think is best?
+
+I like `{py:mod}` because it is semantic and renders like a inline-literal. We don't require link in these cases.
```

Comment:
Thanks for the explanation, and that makes sense to me now. Please delete the todo discussion for this one.